.github/workflows/ci.yml

```diff
@@ -30,29 +26,42 @@ jobs:
     - name: Set display to virtual frame buffer
       run: export DISPLAY=:99.0
 
-
-
-    - name: Install npm dependencies
-      run: npm install
+    - name: Set up test environment
+      run: |
+        sudo apt-get update
+        sudo apt-get install xvfb gettext
+        wget https://github.com/mozilla/geckodriver/releases/download/v0.25.0/geckodriver-v0.25.0-linux64.tar.gz
+        mkdir geckodriver
+        tar -xzf geckodriver-v0.25.0-linux64.tar.gz -C geckodriver
```

Comment:
why are we installing these dependencies? we're not running `bok-choy` tests in this workflow

Reply:
Thats a mistake. I will resolve that